requirements.in

```diff
@@ -7,6 +7,8 @@ django-ilmoitin
 django-parler
 django-cleanup
 django-filter
+django-storages
+django-storages[google]
```

Comment:
if you already install `django-storages` you get all the storage backends and you do not need to separately install `django-storages[google]`

Reply:
Hmn, I don't think it works like that. I tried to install `django-storages` alone and then it complain about some missing gcloud library.

Reply:
hm 🤔

Reply:
https://github.com/jschneier/django-storages/issues/389 It seems to be a known issue. They tried to put the `google-cloud-storage` or any per-storage libraries to `extra_required` libraries. However there is no mention about this in the document

Reply:
Well, there is but it is not highlighted that you **have to** install the dependency

Reply:
yes! you are right. 🙂 you need to specify this option as per [`django-storages`' `setup.cfg`](https://github.com/jschneier/django-storages/blob/master/setup.cfg#L48) 👍